StanwoodCore/Protocols/Wireframe.swift

```diff
@@ -25,16 +25,28 @@
 
 import UIKit
 
-/// :nodoc:
-/*public*/ class Wireframe<M: MetaModule> {
 
-    /// :nodoc:
-    /*public*/ static func makeViewController() -> M.ViewController {
+/// This generic class provide a way of instantiating all the objects needed for a given module.
+///
+/// ```
+/// // Example:
```

Comment:
@lucaangeletti Let's use the correct format for `Example`

https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/Example.html#//apple_ref/doc/uid/TP40016497-CH58-SW1

Reply:
@talezion Unfortunately that markup still does not work with Xcode, it's Playground only